MIGRATING.md

```diff
@@ -0,0 +1,84 @@
+# Okta Signin Widget migration guide
```

Comment:
nit: Signin -> Sign-In

Reply:
nitnit: `Okta Sign-in Widget migration guide`